src/math/welch.js

```diff
@@ -2,6 +2,48 @@ import { periodogram } from './periodogram';
 import { windowApply } from '../data/windowApply';
 import { nextpow2 } from './nextpow2';
 
+const d_mean = (arr) => arr.reduce((a, b) => a + b, 0) / arr.length;
```

Comment:
Nice! Can you add a corresponding unit test to cover this logic in [test/tests/shared/math/welch.test.js](https://github.com/pwstegman/bci.js/blob/master/test/tests/shared/math/welch.test.js)? Maybe something like a copy of the existing unit test but we add a very obvious upward trend to the input

Reply:
Sent a fresh pull request with default 'none'